user/reference/xenial.md

```diff
@@ -141,13 +139,30 @@ repositories.
 | gradle  | 4.10.2  |
 | maven   | 3.5.4   |
 
+## PHP support
+As always, `phpenv` is available for dynamic runtime selection. The following
+PHPs are preinstalled:
+
+| alias | version |
+|:------|:--------|
+| 5.6   | 5.6.36  |
+| 7.1   | 7.1.19  |
+| 7.2   | 7.2.7   |
+
 ## Databases and services
 
-Pre-installed Databases and services:
+Pre-installed Databases and services are:
```

Comment:
How about being more explicit here.

The following services and databases are preinstalled but but do not run by default. To use any of them, add it to the `services` key in your `travis.yml`:

Which also gets rid o ftext before _and_ after the table